src/NServiceBus.Persistence.ServiceFabric.Tests/ComponentTests/Sagas/When_persisting_a_saga_with_no_defined_unique_property.cs

```diff
@@ -12,18 +12,16 @@ public class When_persisting_a_saga_with_no_defined_unique_property : SagaPersis
         {
             configuration.RequiresFindersSupport();
 
-            var sagaId = Guid.NewGuid();
-            var sagaData = new SagaWithoutCorrelationPropertyData
-            {
-                Id = sagaId,
-                FoundByFinderProperty = sagaId.ToString()
-            };
+            var correlationPropertyData = Guid.NewGuid().ToString();
+            var sagaData = new SagaWithoutCorrelationPropertyData { FoundByFinderProperty = correlationPropertyData };
```

Comment:
A bit confusing that a saga w/o correlation property is using a data that is stored with a variable named `correlationPropertyData`